### PR TITLE
Sol3 USB Split features & additional LEDs

### PR DESCRIPTION
**Pull request by Legonut**
## Description
Fixes USB reconnect issues on power cycle for certain use cases and adds an additional 62 LEDs present in the fullhand case upgrade.

## Types of Changes

- [ ] Core
- [X] Bugfix
- [ ] New feature
- [X] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Should fix USB reconnect issues on power cycle for some users.

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/rgbkb/sol3/rev1/config.h [hunk 1]
```diff
@@ -9,6 +9,11 @@
 
 #pragma once
 
+#define SPLIT_WATCHDOG_ENABLE
+#define SPLIT_WATCHDOG_TIMEOUT 3000
+#define SPLIT_USB_TIMEOUT 2000
+#define SPLIT_USB_TIMEOUT_POLL 10
+
```
Comment (zvecr) [inline keyboards/rgbkb/sol3/rev1/config.h:16]: `SPLIT_WATCHDOG_ENABLE` can go in `keyboard.json` as `split.transport.watchdog`, the rest duplicate the defaults
```suggestion
```

Comment (Legonut) [reply keyboards/rgbkb/sol3/rev1/config.h]: @zvecr Thanks!

Comment (drashna) [reply keyboards/rgbkb/sol3/rev1/config.h]: `split.transport.watchdog` was never re-added?

**Review by drashna — APPROVED** [review]

File: keyboards/rgbkb/sol3/rev1/config.h (hunk 1 above)
Comment (zvecr) [reply keyboards/rgbkb/sol3/rev1/config.h]: As this was resolved without a comment, we are to assume that its not needed?

If so, please could you update the PR title and description.

Comment (Doomsdayrs) [reply keyboards/rgbkb/sol3/rev1/config.h]: No it is needed, the usb issues exist.

File: keyboards/rgbkb/sol3/rev1/keyboard.json [hunk 2]
```diff
@@ -258,4 +258,4 @@
             ]
         }
     }
-}
+}
```
Comment (fauxpark) [inline keyboards/rgbkb/sol3/rev1/keyboard.json:261]: Please keep the newline at EOF.

File: keyboards/rgbkb/sol3/rev1/rev1.c [hunk 3]
```diff
@@ -268,4 +284,4 @@ void keyboard_post_init_kb(void) {
 void housekeeping_task_kb(void) {
     touch_encoder_update(TOUCH_ENCODER_SYNC);
     rgb_menu_update(RGB_MENU_SYNC);
-}
+}
```
Comment (fauxpark) [inline keyboards/rgbkb/sol3/rev1/rev1.c:287]: As above.

File: keyboards/rgbkb/sol3/rev1/config.h (hunk 1 above)
Comment (zvecr) [reply keyboards/rgbkb/sol3/rev1/config.h]: Then you need to make the required changes to the `keyboard.json`.

Comment (Doomsdayrs) [reply keyboards/rgbkb/sol3/rev1/config.h]: Tested locally, it works, although the delay between checks is questionable? Possibly needs an error noise.

**Comment by Legonut** [comment]
@drashna I tried adding the split watchdog & USB timeout, but it appears to be having build issues

**Comment by drashna** [comment]
Looks like there are merge conflicts here that are blocking things

**Pull request closed without merging**